stdlib/public/runtime/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ add_swift_library(swiftRuntime OBJECT_LIBRARY TARGET_LIBRARY
   C_COMPILE_FLAGS ${swift_runtime_library_compile_flags}
   LINK_FLAGS ${swift_runtime_linker_flags}
   INSTALL_IN_COMPONENT never_install
-  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX)
+  TARGET_SDKS ANDROID CYGWIN FREEBSD LINUX WINDOWS)
```

Comment:
Let's discuss on-list to keep this pull request focused on just Windows. :) Even if mulle-objc doesn't work 100%, I think it's a good idea to allow Objective-C interop to be turned on or off for any given target.